certbot/latest/letsencrypt-auto-requirements.txt

```diff
@@ -1,202 +1,197 @@
-# Extracted from certbot-auto on 2019-03-06.
-# https://github.com/certbot/certbot/blob/v0.32.0/certbot-auto
+# Extracted from certbot-auto on 2019-04-07.
+# https://github.com/certbot/certbot/blob/v0.34.0/certbot-auto
```

Comment:
This says 0.34.0, but the contents look like 0.34.1? 😋

Reply:
Weird, i though i used the 0.34.0 tag. Let me check

Reply:
Apparently not, 0.34.1 it is :)